docs/zh/latest/plugins/prometheus.md

```diff
@@ -118,12 +118,49 @@ plugin_attr:
 
 ### 可有的指标
 
-* `Status codes`: upstream 服务返回的 HTTP 状态码，每个服务返回状态码的次数或者所有服务的状态码次数总和都可以统计到。
-* `Bandwidth`: 流经apisix的总带宽(可分出口带宽和入口带宽). 每个服务指标或者是所有服务指标的总和都可以统计到。
+* `Status codes`: upstream 服务返回的 HTTP 状态码，每个服务返回状态码的次数或者所有服务的状态码次数总和都可以统计到。具有的维度：
```

Comment:
I think you need add a issue for improve grammar and grammar, the writing skill is not good

Reply:
the doc is ready hard to read.

Reply:
added: [https://github.com/apache/apisix/issues/3689 ](https://github.com/apache/apisix/issues/3689)

Reply:
ping @moonming 

![image](https://user-images.githubusercontent.com/6814606/109386725-abf42b00-7937-11eb-9df8-b4fa54682645.png)